init_x: 0  # initial x-pos 
init_y: 0  # initial y-pos
init_z: 110  # initial z-pos
init_psi: 0  # initial heading

max_omega: 5.0  # drone max angular velocity
max_zvel: 5  # drone max vertical velocity

agent_l: 3  # agent length for visualization

hvel: 60  # agent x-y velocity  
vvel: 2  # agent z-velocity

# number of random targets 
n_rand_targets: -1  # -1 if providing custom list of targets 

del_t: .02  # timestep between sim updates

waypoint_threshold: 3

K_p: 10  # PID angular proportionality constant
K_p_z: 0.01  # PID z-axis proportionality constant

targets:  # start ship IDs at 1. angles in radians. speed in m/s  # to use random targets, se this to empty list
  - {id: 1, x: 600, y: -700, heading: 1.57, linear_speed: 5, angular_speed: 0, linear_speed_std: 0.0, angular_speed_std: 0.00}
  - {id: 2, x: 0, y: -800, heading: 1.57, linear_speed: 5, angular_speed: 0, linear_speed_std: 0.0, angular_speed_std: 0.00}
  # - {id: 2, x: 600, y: -150, heading: 1.54, linear_speed: 0.2, angular_speed: 0, linear_speed_std: 0.2, angular_speed_std: 0.01}
  - {id: 3, x: 100, y: -1800, heading: 1.57, linear_speed: 5, angular_speed: 0.0, linear_speed_std: 0.0, angular_speed_std: 0.00}
  - {id: 4, x: 1200, y: -1000, heading: 1.57, linear_speed: 5, angular_speed: 0.0, linear_speed_std: 0.0, angular_speed_std: 0.00}
  - {id: 5, x: 1400, y: -1200, heading: 1.57, linear_speed: 5, angular_speed: 0.0, linear_speed_std: 0.0, angular_speed_std: 0.00}
